saleor/core/utils/__init__.py

```diff
@@ -130,3 +131,28 @@ def create_superuser(credentials):
     else:
         msg = 'Superuser already exists - %(email)s' % credentials
     return msg
+
+
+@python_2_unicode_compatible
+class TranslationWrapper(object):
+
+    def __init__(self, instance, locale):
+        self._instance = instance
+        self._locale = locale
+        self._translation = instance.translations.filter(language_code=locale).first()
```

Comment:
This is guaranteed to execute a database query even if translations were properly prefetched.